### ISSUE TITLE
Fatal error: Cannot redeclare App\Livewire\Dashboard\Brands\Forms\UpdateDetailForm::$brand

### ISSUE THREAD

**Issue by singhpk89** [VemtoOrg/vemto2-issues#146]
**Error Message:**
(error) FAILED to execute command: "php" "/Applications/Vemto.app/Contents/Resources/static/VMTTL1"
    |> 
Fatal error: Cannot redeclare App\Livewire\Dashboard\Brands\Forms\UpdateDetailForm::$brand in /Users/prakashsingh/Documents/Projects/y2024/Raymond/app/Livewire/Dashboard/Brands/Forms/UpdateDetailForm.php on line 13

    |> PHP Fatal error:  Cannot redeclare App\Livewire\Dashboard\Brands\Forms\UpdateDetailForm::$brand in /Users/prakashsingh/Documents/Projects/y2024/Raymond/app/Livewire/Dashboard/Brands/Forms/UpdateDetailForm.php on line 13

**Stack:**
```
Error: Command failed: "php" "/Applications/Vemto.app/Contents/Resources/static/VMTTL1"
PHP Fatal error:  Cannot redeclare App\Livewire\Dashboard\Brands\Forms\UpdateDetailForm::$brand in /Users/prakashsingh/Documents/Projects/y2024/Raymond/app/Livewire/Dashboard/Brands/Forms/UpdateDetailForm.php on line 13

    at ChildProcess.exithandler (node:child_process:407:12)
    at ChildProcess.emit (node:events:527:28)
    at maybeClose (node:internal/child_process:1092:16)
    at Socket.<anonymous> (node:internal/child_process:451:11)
    at Socket.emit (node:events:527:28)
    at Pipe.<anonymous> (node:net:709:12)
```

**How to reproduce:**
1. 
2. 
3. 

**Expected behavior:**

**Actual behavior:**

**Additional context:**

---

Vemto 2.0.14
darwin 22.5.0
Locale: en-GB
App version: 2.0.14

**Comment by TiagoSilvaPereira** [comment]
@singhpk89 seems you have the variable $brand declared twice in your UpdateDetailForm class

**Comment by TiagoSilvaPereira** [comment]
Fixed on 2.0.24
